### PR TITLE
Burak-caps-text

### PR DESCRIPTION
**Pull request by myniqx**
https://github.com/freedomcombination/monorepo/issues/737

image

image

image

image

### CODE REVIEW

**Review by 7alip — CHANGES_REQUESTED** [review]
Thanks.

The Caps component is displayed on the dashboard and TrendRights cards. However, on Twitter, users will see the SEO OpenGraph image, which is more important.

This style improvement should also be applied to `packages/services/src/api/og.tsx`.

To test the og image path, use the '' component with the ogParams query.

**Comment by myniqx** [comment]
i updated to releated changes, is there any way to check is it working correctly ?

File: packages/services/src/api/og.tsx [hunk 1]
```diff
@@ -230,6 +230,9 @@ export const ogRouter = async (req: NextRequest) => {
                 fontSize: `${dimensions.font}px`,
                 overflow: 'hidden',
                 whiteSpace: 'pre-wrap',
+                textAlign: 'center',
```
Comment (7alip) [inline packages/services/src/api/og.tsx:233]: @myniqx Shall we make it not center but left aligned? It looks not good if the last line has few words.

**Review by 7alip — APPROVED** [review]
Thanks

File: packages/services/src/api/og.tsx (hunk 1 above)
Comment (myniqx) [reply packages/services/src/api/og.tsx]: what about this ?
`textAlign: flip ? 'right' : 'left',`

**Review by 7alip — APPROVED** [review]
Thanks

File: packages/services/src/api/og.tsx (hunk 1 above)
Comment (7alip) [reply packages/services/src/api/og.tsx]: Let's keep it like this. We can update this logic when we start implementing new templates

**Pull request merged**